contracts/smart-account/modules/AccountRecoveryModule.sol

```diff
@@ -34,11 +34,13 @@ contract AccountRecoveryModule is
     string public constant NAME = "Account Recovery Module";
     string public constant VERSION = "0.1.0";
 
-    // keccak256(abi.encodePacked("ACCOUNT RECOVERY GUARDIAN SECURE MESSAGE"))
-    bytes32 public constant CONTROL_HASH =
-        0x1c3074d7ce4e35b6f0a67e3db0f31ff117ca8f0a1853e86a95837fea35af828b;
-    bytes32 public immutable controlHashEthSigned =
-        CONTROL_HASH.toEthSignedMessageHash();
+    // execute(address,uint256,bytes)
+    bytes4 public constant EXECUTE_SELECTOR = 0xb61d27f6;
+    // execute_ncC(address,uint256,bytes)
+    bytes4 public constant EXECUTE_OPTIMIZED_SELECTOR = 0x0000189a;
+    // Hash to be signed by guardians to make a guardianId
+    string public constant CONTROL_MESSAGE =
+        "ACCOUNT RECOVERY GUARDIAN SECURE MESSAGE";
```

Comment:
Making the message fit to  `32bytes` saves gas on deployment (~2200 gas) and on `abi.encodePacked` (~80-100 gas)

Reply:
done